mappings/net/minecraft/block/Block.mapping

```diff
@@ -215,69 +294,78 @@ CLASS net/minecraft/class_17 net/minecraft/block/Block
 		ARG 3 y
 		ARG 4 z
 		ARG 5 player
-	METHOD method_1609 (Lnet/minecraft/class_18;IIII)V
+	METHOD method_1609 neighborUpdate (Lnet/minecraft/class_18;IIII)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 		ARG 5 id
-	METHOD method_1610 onHit (Lnet/minecraft/class_18;IIILnet/minecraft/class_54;)V
+	METHOD method_1610 onBlockBreakStart (Lnet/minecraft/class_18;IIILnet/minecraft/class_54;)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 		ARG 5 player
-	METHOD method_1611 (Lnet/minecraft/class_18;III)V
+	METHOD method_1611 onPlaced (Lnet/minecraft/class_18;III)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1612 (Lnet/minecraft/class_18;IIII)V
+	METHOD method_1612 onMetadataChange (Lnet/minecraft/class_18;IIII)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1613 (Lnet/minecraft/class_18;III)V
-		ARG 1 world
-	METHOD method_1614 (Lnet/minecraft/class_18;IIILnet/minecraft/class_127;)V
+		ARG 5 meta
+	METHOD method_1613 onDestroyedByExplosion (Lnet/minecraft/class_18;III)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_1615 (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;)V
+	METHOD method_1614 onPlaced (Lnet/minecraft/class_18;IIILnet/minecraft/class_127;)V
 		ARG 1 world
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
+		ARG 5 placer
+	METHOD method_1615 applyVelocity (Lnet/minecraft/class_18;IIILnet/minecraft/class_57;)V
```

Comment:
Mistakenly named `applyVelocity` when it was suggested to be `onEntityCollision` in previous review (also wrong argument names)